composer.json

```diff
@@ -12,7 +12,8 @@
     "php": ">=7.1",
     "spryker/store": "^1.1.0",
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose Yes now